.claude/skills/eval.md

```
@@ -0,0 +1,125 @@
+# Eval
+
+Run LLM evaluations from the examples directory.
+
+## Usage
+
+`/eval [example-name] [options]`
+
+## Options
+
+| Option | Description |
+|--------|-------------|
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)